Content.Server/Maps/GameMapManager.cs

```diff
@@ -15,29 +16,38 @@ public sealed class GameMapManager : IGameMapManager
     [Dependency] private readonly IConfigurationManager _configurationManager = default!;
     [Dependency] private readonly IPlayerManager _playerManager = default!;
     [Dependency] private readonly IRobustRandom _random = default!;
-    [Dependency] private readonly IChatManager _chatManager = default!;
-
-    [ViewVariables]
+    
+    [ViewVariables(VVAccess.ReadOnly)]
     private readonly Queue<string> _previousMaps = new();
-    [ViewVariables]
-    private GameMapPrototype _currentMap = default!;
-    [ViewVariables]
-    private bool _currentMapForced;
-    [ViewVariables]
+    [ViewVariables(VVAccess.ReadOnly)]
+    private GameMapPrototype? _configSelectedMap = default!;
```

Comment:
There's no need to null suppress this and it gives the wrong impression, same with below.